git-hooks/pre-commit.sh

```diff
@@ -16,7 +16,7 @@ cr='
 '
 for f in $(git diff --cached --name-only | grep -E $FILES_PATTERN)
 do
-  MATCH=`grep --invert-match --no-messages $REQUIRED $f | head -n1`
+  MATCH=`grep -L $REQUIRED $f | head -n 1`
```

Comment:
So I think that this looks for the required text `ANSIBLE_VAULT` in the file `$f` and if it isn't found in a file then it prints the file's name.

Previously it was returning file lines, so it was returning every line except for the first one of the vault file